hist/hist/src/TGraph.cxx

```diff
@@ -2568,6 +2556,25 @@ void TGraph::SwapPoints(Int_t pos1, Int_t pos2)
    SwapValues(fY, pos1, pos2);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Update the fX and fY arrays with the sorted values.
+
+void TGraph::UpdateArrays(std::vector<Int_t> sorting_indices, Int_t numSortedPoints, Int_t low)
```

Comment:
maybe it would be better to mark it as a constant reference to avoid copying if the vector is large? Or otherwise, just marking it as "const".

Reply:
Indeed, the signature should be 
```suggestion
void TGraph::UpdateArrays(const std::vector<Int_t> &sorting_indices, Int_t numSortedPoints, Int_t low)
```

For `TGraph` and all derived classes